miscnn/neural_network/model.py

```diff
@@ -95,17 +98,20 @@ def build_model(self, architecture):
         # Assemble the input shape
         input_shape = (None,)
         # Initialize model for 3D data
+        
+        patch_handler = self.preprocessor.patch_handler
```

Comment:
maybe renaming it to image_handler due to it handles not only patches but also full images?

Reply:
vice versa can full images be considered trivial patches. Also image handler seems rather unspecific.

Reply:
Thinking about renaming.